cms/envs/devstack.py

```diff
@@ -295,3 +290,9 @@ def should_show_debug_toolbar(request):  # lint-amnesty, pylint: disable=missing
 #################### Network configuration ####################
 # Devstack is directly exposed to the caller
 CLOSEST_CLIENT_IP_FROM_HEADERS = []
+
+################# New settings must go ABOVE this line #################
+########################################################################
+# See if the developer has any local overrides.
+if os.path.isfile(join(dirname(abspath(__file__)), 'private.py')):
+    from .private import *  # pylint: disable=import-error,wildcard-import
```

Comment:
Moved this to the bottom of the file so that settings in `private.py` aren't accidentally overwritten.